packages/jellyfish-api-core/__tests__/category/token/createToken.test.ts

```diff
@@ -1,6 +1,7 @@
 import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
 import { ContainerAdapterClient } from '../../container_adapter_client'
 import waitForExpect from 'wait-for-expect'
+import { BigNumber } from '@defichain/jellyfish-api-core'
```

Comment:
`import BigNumber from 'bignumber.js'`
Ahmed, Bella, AikChun, and I have changed to this way to import BigNumber